tests/phpunit/includes/Fake_Site_Connection_Trait.php

```diff
@@ -20,52 +20,84 @@
 trait Fake_Site_Connection_Trait {
```

Comment:
Changes to this trait were not _strictly_ necessary but I think this is a worthwhile enhancement that makes the implementation a bit cleaner while adding methods for getting credentials without "applying" them.